frontend/src/static/js/components/webstatus-overview-cells.ts

```diff
@@ -473,10 +474,36 @@ export function renderGroupsRow(columns: ColumnKey[]): TemplateResult {
   `;
 }
 
+export function renderBookmarkHeaderCells(
+  bookmarkName: string,
+  columns: ColumnKey[],
+): TemplateResult[] {
+  const headerCells: TemplateResult[] = columns.map(col => {
+    if (col == ColumnKey.Name) {
+      const title = `Sorted by ${bookmarkName} query order`;
+      return html`${renderUnsortableHeaderCell(col, title)}`;
+    } else {
+      return html`${renderUnsortableHeaderCell(col)}`;
+    }
+  });
+  return headerCells;
+}
+
 export function renderHeaderCell(
   routerLocation: {search: string},
   column: ColumnKey,
   sortSpec: string,
+): TemplateResult {
+  if (CELL_DEFS[column].unsortable) {
+    return renderUnsortableHeaderCell(column);
+  }
+  return renderSortableHeaderCell(routerLocation, column, sortSpec);
+}
+
+export function renderSortableHeaderCell(
```

Comment:
We can un-export this for now.

```suggestion
function renderSortableHeaderCell(
```